master/buildbot/www/oauth2.py

```diff
@@ -248,7 +251,8 @@ def __init__(self,
                     'Retrieving team membership information using GitHubAuth is only '
                     'possible using GitHub api v4.')
         else:
-            self.apiResourceEndpoint = self.serverURL + '/graphql'
+            defaultGraphqlEndpoint = self.serverURL + '/graphql'
+            self.apiResourceEndpoint = enterpriseGraphqlEndpoint or defaultGraphqlEndpoint
```

Comment:
you can define default self.apiResourceEndpoint line 228 and then overwrite it line 237.

Reply:
got it